scripts/systemd/bumblebeed.service.in

```diff
@@ -3,6 +3,7 @@ Description=Bumblebee C Daemon
 
 [Service]
 Type=simple
+ExecStartPre=/usr/bin/prime-select intel
```

Comment:
This should be:

```
ExecStartPre=-/usr/bin/prime-select intel
```

As prime-select is only available on certain distributions, not everywhere.